packages/autorest.python/autorest/codegen/__init__.py

```diff
@@ -122,7 +122,7 @@ def remove_cloud_errors(yaml_data: Dict[str, Any]) -> None:
     @staticmethod
     def _build_package_dependency() -> Dict[str, str]:
         return {
-            "dependency_azure_mgmt_core": "azure-mgmt-core<2.0.0,>=1.3.0",
+            "dependency_azure_mgmt_core": "azure-mgmt-core<2.0.0,>=1.3.2",
```

Comment:
latest autorest.python needs `azure-core>=1.24.0` and only `azure-mgmt-core>=1.3.2` meet the requirements: https://github.com/Azure/azure-sdk-for-python/pull/25103/files